tests/unit/amo/components/TestRatingsByStar.js

```diff
@@ -218,25 +246,38 @@ describe(__filename, () => {
       addon,
       location: createFakeLocation({ query: { src } }),
     });
-    const counts = root.find('.RatingsByStar-count').find(Link);
+    const tableRow = root.find('.RatingByStar-table-row');
 
-    function validateLink(link, score) {
-      expect(link).toHaveProp(
-        'to',
-        reviewListURL({ addonSlug: addon.slug, score, src }),
+    const clientApp = CLIENT_APP_ANDROID;
+    const lang = 'en-US';
+
+    function validateLink(row, score) {
+      expect(row).toHaveProp('onClick');
+
+      row.simulate('click', createFakeEvent({}));
+
+      sinon.assert.calledWithExactly(
+        fakeHistory.push,
+        `/${lang}/${clientApp}${reviewListURL({
+          addonSlug: addon.slug,
+          score,
+          src,
+        })}`,
       );
     }
 
-    validateLink(counts.at(0), '5');
-    validateLink(counts.at(1), '4');
-    validateLink(counts.at(2), '3');
-    validateLink(counts.at(3), '2');
-    validateLink(counts.at(4), '1');
+    validateLink(tableRow.at(0), '5');
+    validateLink(tableRow.at(1), '4');
+    validateLink(tableRow.at(2), '3');
+    validateLink(tableRow.at(3), '2');
+    validateLink(tableRow.at(4), '1');
   });
 
   it('renders IconStar', () => {
     const root = render();
-    const star = root.find(IconStar).at(0);
```

Comment:
Won't this still work? Are the changes to this test required?

Reply:
Yes, it is working fine without any changes. I will update that.